test/index.js

```diff
@@ -105,6 +105,27 @@ describe('wt-tracker.', () => {
       done();
     });
   });
+  it('should handle after first load events', (done) => {
+    const events = [{hello: 'world', url: HREF}];
+    let touchedCount = 0;
+    wt('resetFirstLoad');
+    wt('afterFirstLoad', () => {
+      touchedCount ++;
+    });
+    assert.equal(touchedCount, 0);
+    runEvents(events, () => {
+      assert.equal(touchedCount, 1);
+      wt('afterFirstLoad', () => {
+        touchedCount ++;
+      });
+      assert.equal(touchedCount, 2);
+      done();
```

Comment:
If I understand correctly, we're using `done()` essentially to indicate that the promise should be resolved - is that correct?

Reply:
it's an async test - line `108` refers to argument `done`